action.yml

```diff
@@ -1,108 +1,19 @@
-name: Pros Build
-description: Automatically build pros projects
+name: 'ABUCKY0 PROS Build'
```

Comment:
Why put your name on this?

Reply:
Because GitHub wouldn’t let me publish the action otherwise. I’ll fix that in a bit